src/component/dice/Dice.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+
+
+class Dice extends Component {
+    constructor(props){
+        super(props)
+        this.state={
+            diceImg:''
+        }
+    }
+    
+    getRandomImage = () => {
+        const diceArray = [
+                '/img/dice/dice1.png',
+                '/img/dice/dice2.png',
+                '/img/dice/dice3.png',
+                '/img/dice/dice4.png',
+                '/img/dice/dice5.png',
+                '/img/dice/dice6.png']
+        
+        const randomNumber = Math.floor(Math.random() * (5 - 0 + 1)) + 0
```

Comment:
Kinda confused with your math: (5-0+1)=6, so Math.floor(Math.random() * 6) does the job just fine xD. 

Besides your hate for math, everything looks great xD